Google.Api.Gax.Grpc/Rest/RestChannel.cs

```diff
@@ -104,11 +104,15 @@ internal sealed class RestChannel : ChannelBase
             Func<Status> statusFunc = () => GetStatus(statusResponseSource.Task);
             Func<Metadata> trailersFunc = () => GetTrailers(statusResponseSource.Task);
 
+            // The response stream will dispose of the cancellation context if:
+            // - The user disposes of it directly
```

Comment:
There should be an extra line in this comment, not a replacement one. Because the caller gets the PartialDecodingStreamReader directly, and can dispose it. (They'd need to cast to IAsyncDisposable or IDisposable, but there are cases where that happens.)